src/org/usfirst/frc2175/commandmapper/JoystickEventMapper.java

```diff
@@ -27,6 +29,14 @@ public JoystickEventMapper(RobotConfig robotConfig,
         retractCatapult
                 .whenPressed(new RetractCatapultCommand(robotSubsystems));
 
+        JoystickButton runIntakeIn = gamepadConfig.getIntakeInButton();
+        runIntakeIn.whenPressed(new RunIntakeInGroup(robotSubsystems,
+                robotConfig.getIntakeConfig()));
```

Comment:
I should say that yes, your code does work! It's just that we can clean it up a little.